locales/en-US/messages.properties

```diff
@@ -102,6 +102,13 @@ denotes_required_field=denotes required field
 share_a_link_with_your_fellowers=Share a link with your followers
 i_donated_to_mozilla=I donated to @mozilla today because I #lovetheweb. Join me and help build + protect the Web forever
 we_include_you=We want to include you in our work!
+These are the variants:
```

Comment:
Sorry, already merged :(

I saw that, and thought it was kinda odd, but I'm not super familiar with .properties syntax and thought maybe it could handle it without the =